Extensions/ServiceNow/Src/Tasks/UpdateChangeRequest/UpdateChangeRequestV2/task.json

```diff
@@ -126,15 +127,30 @@
             "Expression": "eq(count(root['result']), 1)"
           },
           "ExecutionOptions": {
-            "OutputVariables": "{\"CHANGE_CORRELATION_ID\" :  \"jsonpath('$.result[0].correlation_id')[0]\"}"
```

Comment:
Are we going to have ChangeRequestNumber input as required?

Reply:
Yeah, keeping it as required for release functionality purposes. Users can give or have in empty in YAML.